frontend/src/components/translation/TranslationTable.tsx

```diff
@@ -95,7 +95,7 @@ const TranslationTable = ({
           </Tooltip>
         )}
         <Flex direction="column" width="140px" margin="5px">
-          {reviewPage ? (
+          {inReviewStage ? (
```

Comment:
I dont like this prop name because it implies that when the translation is in-Review-Stage, the `StatusBadge` will display in the translation platform even though the platform is supposed to be disabled. 
I _know_ this is not the actual behavior of the application since you only set `inReviewStage` from the Review Page, but this prop name should be different. What do you think of continuing to use `editable` exclusively for the cells, and introducing `disabled` or `reviewFeatures` or `isReviewable`  to be used for additional features in the TranslationTable (eg StatusBadge, Comment button, Approve All)
open to other name ideas!